.ado/publish.yml

```diff
@@ -45,25 +45,15 @@ jobs:
 
       variables:
         DotNetMoniker: net8.0
-        # Temporary variables to collect CodeQL info to debug OSX ARM64 build issue.
-        # They must be removed as soon as they not needed to save storage.
+        # Temporary disable CodeQL for OSX ARM64.
+        # It must be restored after CodeQL fixes the issue where they install x64 bits on OSX ARM64.
         ${{ if eq( matrixEntry.TargetRuntime, 'osx-arm64' ) }}:
-          Codeql.PublishDatabaseLog: true
-          Codeql.Cadence: 0
+          Codeql.Enabled: false
```

Comment:
I didn't think this would fix this issue because from what I read the injection still happens, it just returns immediately. Since the problem is with loading the injection, I think it will still fail.

Reply:
Hmm. Let me still try it. This fix was suggested by CodeQL team today.

Reply:
The CI build is passed. So, the fix works. :)